backend/account/query.py

```diff
@@ -15,9 +15,9 @@ def get_or_create_user(payload: dict) -> User:
         return User.objects.create(
             username=User.objects.all().count(),
             google_id=payload["sub"],
-            email=payload["email"] if payload.get("email") else "No Email Found",
-            image_url=payload["picture"] if payload.get("picture") else None,
-            first_name=payload["given_name"] if payload.get("given_name") else "No First Name Found",
+            email=payload["email"] if payload.get("email") else "",
+            image_url=payload["picture"] if payload.get("picture") else "localhost:8000/static/logo-compact.png",
```

Comment:
Is there a way to reference the image without using `localhost:8000`? This will break when we have a production build that isn't running on localhost, so we should reference a local file directly, or pick an online image.